AmpersandData/SystemContext/Interfaces.ifc

```diff
@@ -10,9 +10,9 @@ CONTEXT PrototypeFramework IN ENGLISH
 
     INTERFACE "Edit interface" FOR Administrator : I[PF_Interface] BOX
         [ "Interface"           : I
-        , "Is public"           : isPublic      cRud -- do not allow editing, access control of interfaces is not dynamic (yet)
+        , "Is public"           : isPublic      cRUd -- do not allow editing, access control of interfaces is not dynamic (yet)
```

Comment:
This comment should be modified as well, I suppose

Reply:
Of course. I’ll push a change for that